docs/airdrop-eligibility-distribution.md

```diff
@@ -191,11 +190,10 @@ To calculate the number of tokens that you're eligible to claim, first determine
 
 Scroll to the section that describes your scenario and follow the instructions.
 
-
 #### Scenario 1: All of your qualifying actions occurred before Nitro was launched.
 
- 1. First, refer to the [qualifying actions section below](#qualifying-actions) to determine how many points you earned on Arbitrum One. Because Arbitrum Nova was launched after Arbitrum Nitro, you won't have any qualifying actions on Arbitrum Nova.
- 2. Next, refer to the [point-to-token conversion table below](#point-to-token-conversion-table) and determine how many tokens you're eligible to claim based on the number of points you earned.
+ 1. First, refer to the [qualifying actions section](#qualifying-actions) to determine how many points you earned on Arbitrum One. Because Arbitrum Nova was launched after Arbitrum Nitro, you won't have any qualifying actions on Arbitrum Nova.
+ 2. Next, refer to the [point-to-token conversion table](#point-to-token-conversion-table) and determine how many tokens you're eligible to claim based on the number of points you earned.
```

Comment:
Just to minimize back-and-forth and get this published sooner rather than later, would you mind submitting a draft procedure that includes examples as a commit suggestion or PR that articulates the precise business logic from your perspective?

This would ideally include the edge cases (RE sybils and timing) that we want our users to be aware of as they follow the steps.

I can then provide editorial review on your draft, asking specific questions as needed, instead of trying to guess what that "similar logic" looks like :p. 

I have notifications enabled, will review once the precise logic is drafted in the format of user-facing instructions. Thank you sir!

Reply:
[marking as resolved - let's move outstanding todos / clarifications to the [v2 PR](https://github.com/ArbitrumFoundation/docs/pull/230)]